.github/workflows/generatemd.yml

```diff
@@ -0,0 +1,66 @@
+name: Generate Proposal MD
+
+on:
+  workflow_dispatch: #periodic
+    inputs:
+      logLevel:
```

Comment:
@bimsarapathiraja This input is not used right? Shall we remove in that case?

Reply:
removed